docs/var.tfvars-doc.md

```diff
@@ -224,6 +224,12 @@ volume_size                 = "300" # Value in GB
 volume_storage_template     = ""
 ```
 
+If you need to attach additional data volumes to the cluster nodes use the following variables.
```

Comment:
@aishwaryabk  Need to update the Docs, this needs to be removed and added as part of master and worker node vars (with proper var names)